utils/certs/gen_certificates.sh

```diff
@@ -167,21 +172,40 @@ function generate_server_cert () {
     ${CERTS}/server.crt"
 }
 
+function populate_clients_dir () {
+    cp "${CERTS}/admin.crt"       "${CLIENTS}/admin.crt"
+    chown daos_server.daos_server "${CLIENTS}/admin.crt"
+    chmod 640                     "${CLIENTS}/admin.crt"
+    cp "${CERTS}/agent.crt"       "${CLIENTS}/agent.crt"
+    chown daos_server.daos_server "${CLIENTS}/agetn.crt"
+    chmod 640                     "${CLIENTS}/agent.crt"
```

Comment:
No reason for these to be unreadable to the public. These are public keys so they are OK for anyone to read.

Reply:
except that our code is very picky about file permissions, and if i recall correctly it doesn't allow world-readable certs.  i agree this is un-necessary and should be relaxed.  and just spotted a typo - agetn...

Reply:
checked that 644 is accepted, changed to 644